### PR TITLE
[RFR] Create a Makefile to install and run the project

### PR DESCRIPTION
**Pull request by adguernier**
[Capture vidéo du 18-04-2023 11:29:27.webm](https://user-images.githubusercontent.com/7949510/232735435-7f6af7b5-e518-45bc-8c7f-f60a0888fad7.webm)
- [x] create a Makefile with useful commands
- [x] enhance the README

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.PHONY: help install run
```
Comment (slax57) [inline Makefile:1]: not found of this, you should only print the help by default IMO

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	yarn install
+
+run: ## Run the game with Yarn
+	yarn start
+
+volta_install: ## Install NodeJS dependencies with Volta and Yarn 
+	volta run yarn install
+
+volta_run: ## Run the game with Volta and Yarn
+	volta run yarn start
```
Comment (slax57) [inline Makefile:20]: You should avoid including make targets that are specific to your setup, and instead configure your dev environment to be compatible with the generic commands

File: Readme.md [hunk 3]
```diff
@@ -12,12 +12,22 @@ Play the game of Kamon on the console
 yarn
 ```
 
+Also, you can run make 
+```sh
+make install
+```
```
Comment (slax57) [inline Readme.md:18]: These instructions should replace the existing IMO. Do not fear to delete things from the initial project. It's your project now, so feel free to change it as you wish :slightly_smiling_face:

File: Readme.md [hunk 4]
```diff
@@ -12,12 +12,22 @@ Play the game of Kamon on the console
...
 ### Playing
 
 ```sh
 yarn start
 ```
 
+Also, you can run make 
+```sh
+make run
+```
```
Comment (slax57) [inline Readme.md:29]: Same

**Review by slax57 — CHANGES_REQUESTED** [review]
Also, this branch needs rebase.

You can keep track of this kind of tasks using the 'checklist' feature of github.
Example:

- [ ] Rebase after #2 has been merged

File: Makefile (hunk 1 above)
Comment (adguernier) [reply Makefile]: Done

File: Makefile (hunk 2 above)
Comment (adguernier) [reply Makefile]: Done

File: Readme.md (hunk 3 above)
Comment (adguernier) [reply Readme.md]: Done

File: Readme.md (hunk 4 above)
Comment (adguernier) [reply Readme.md]: Done

**Review by slax57 — CHANGES_REQUESTED** [review]
One last change I forgot to request: you should mention in the readme that the project requires `yarn` in addition to NodeJS. And then we should be good to go :slightly_smiling_face:

**Review by slax57 — APPROVED** [review]

**Pull request merged**